winui/Scheduler/Month-View.md

```diff
@@ -260,6 +260,24 @@ this.Schedule.MonthViewSettings.TrailingDaysVisibility = Visibility.Collapsed;
 
 ![customize-leading-and-trailing-days-visibility-in-winui-scheduler-month-view](Month-View_Imges/customize-leading-and-trailing-days-visibility-in-winui-scheduler.png)
 
+## Customizing the Number of Visible Weeks
```

Comment:
Move to below [Appointment display count](https://help.syncfusion.com/winui/scheduler/month-view#appointment-display-count)